README.adoc

```diff
@@ -27,52 +27,48 @@ NOTE: This repository contains the guide documentation source. To view the guide
 
 Explore how to enable and customize tracing of JAX-RS and non-JAX-RS methods by using MicroProfile OpenTracing and the Zipkin tracing system.
 
-
 :inv-url: http://localhost:9081/inventory/systems
 :inv-url-docker: http://localhost:9080/inventory/systems
 :sys-url: http://localhost:9080/system/properties
 :zipkin-url: http://localhost:9411
 
-
 == What you'll learn
 
-You will learn how to enable automatic tracing for JAX-RS methods as well as create custom tracers
+You'll learn how to enable automatic tracing for JAX-RS methods as well as create custom tracers
 for non-JAX-RS methods by using MicroProfile OpenTracing.
 
 OpenTracing is a standard API for instrumenting microservices for distributed tracing. Distributed
 tracing helps troubleshoot microservices by examining and logging requests as they propagate through a
-distributed system, allowing developers to tackle the otherwise difficult task of debugging these requests.
-Without a distributed tracing system in place, analyzing the workflows of operations becomes difficult,
-particularly in regard to pinpointing when and by whom a request is received, as well as when a response
-is sent back.
+distributed system. This allows developers to tackle the otherwise difficult task of debugging these requests.
```

Comment:
This allows developers to tackle the otherwise difficult task of debugging these requests.
--->
Distributed tracing allows developers to tackle the otherwise difficult task of debugging these requests.